WcaOnRails/spec/models/competition_wcif_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+ONE_MINUTE_IN_CENTISECONDS = 60*100
+
+RSpec.describe "Competition WCIF" do
+  let!(:competition) {
+    FactoryGirl.create(
+      :competition,
+      :with_delegate,
+      id: "TestComp2014",
+      name: "Test Comp 2014",
+      start_date: "2014-02-03",
+      end_date: "2014-02-05",
+      external_website: "http://example.com",
+      showAtAll: true,
+      event_ids: %w(333 444),
+    )
+  }
+  let(:delegate) { competition.delegates.first }
+  let(:sixty_second_2_attempt_cutoff) { Cutoff.new(numberOfAttempts: 2, attemptValue: ONE_MINUTE_IN_CENTISECONDS) }
+  let(:top_16_advance) { AdvanceToNextRoundRequirement.new(type: "ranking", ranking: 16) }
+  let!(:round333_1) { FactoryGirl.create(:round, competition: competition, event_id: "333", number: 1, cutoff: sixty_second_2_attempt_cutoff, advance_to_next_round_requirement: top_16_advance) }
+  let!(:round333_2) { FactoryGirl.create(:round, competition: competition, event_id: "333", number: 2) }
+  let!(:round444_1) { FactoryGirl.create(:round, competition: competition, event_id: "444", number: 1) }
+  before :each do
+    competition.reload
+  end
+
+  context "to_wcif" do
```

Comment:
Using describe for a particular entity makes sense, I honestly am still very lazy when naming and organizing tests. Thanks for continuing to call me out on it!

I intentionally wrote `RSpec.describe "Competition WCIF"` instead of `RSpec.describe Competition do...context "WCIF"` just to keep the level of indentation in the file down. I do realize this breaks pattern with the rest of the files in our spec/models directory, so I'm willing to change this if someone feels strongly.